projects/m1/018-roulette-payouts/solutions/stefanocroci/sol_018_roulette_payouts.py

```diff
@@ -0,0 +1,43 @@
+# I made sure that if 37 comes out it's considered as 00
+
+# Import generator of random number from 0 to 37
+import random
+num = random.randint(0, 37)
+
+# Define red and black number
+red = [1, 3, 5, 7, 9, 12, 14, 16, 18, 19, 21, 23, 25, 27, 30, 32, 34, 36]
+black = [2, 4, 6, 8, 10, 11, 13, 15, 17, 20, 22, 24, 26, 28, 29, 31, 33, 35]
```

Comment:
actually there is a pattern here.

If you take a look, odd numbers from 1 to 10 are red, even numbers from 11 to 20 are red and so on.

You can avoid to use hardcoded arrays